cpp/src/slice2php/Main.cpp

```diff
@@ -1550,7 +1550,7 @@ compile(const vector<string>& argv)
                         if (!out)
                         {
                             ostringstream os;
-                            os << "cannot open`" << file << "': " << IceInternal::errorToString(errno);
+                            os << "cannot open'" << file << "': " << IceInternal::errorToString(errno);
```

Comment:
Same here, missing space:
```suggestion
                            os << "cannot open '" << file << "': " << IceInternal::errorToString(errno);
```

Reply:
Fixed.